lib/parameters.py

```diff
@@ -308,6 +338,7 @@ def gen_avc_max_frame_size_parameters(spec, profiles):
 gen_hevc_forced_idr_parameters = gen_avc_forced_idr_parameters
 gen_hevc_intref_lp_parameters = gen_avc_intref_lp_parameters
 gen_hevc_max_frame_size_parameters = gen_avc_max_frame_size_parameters
+gen_hevc_pict_lp_parameters = gen_hevc_pict_parameters
```

Comment:
I know I suggested this alias at first, but later realized you have two variants that need handled, `pict` and `pict_lp`, so my first suggestion is wrong, hence the second suggestion/fix... sorry about the confusion from my end.

In your internal test config patches, you use `pict` and `pict_lp` variant keys to define test cases.  But this alias only parses the `pict` variant.  Thus, `pict_lp` is never used from the test config.  We should use separate variants and thus will need two separately defined param gen methods.  Perhaps the common code can be refactored and shared in one function, then just pass the lookup key depending on which variant it's for.

Reply:
It's okay. I modified this part.